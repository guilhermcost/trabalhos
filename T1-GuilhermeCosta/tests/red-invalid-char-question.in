// Testa caracter invalido ?
a - 10?
print a;


