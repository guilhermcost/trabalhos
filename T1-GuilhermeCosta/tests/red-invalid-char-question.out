(2,ID,"a")
(2,SYM,"-")
(2,NUM,"10")
(2,ERROR,"?")
(3,ID,"print")
(3,ID,"a")
(3,SYM,";")
